server/app/services/program/ProgramService.java

```diff
@@ -95,7 +98,9 @@ ErrorAnd<ProgramDefinition, CiviFormError> createProgramDefinition(
       String defaultDisplayName,
       String defaultDisplayDescription,
       String externalLink,
-      String displayMode);
+      String displayMode,
+      Boolean isCommonIntakeForm,
```

Comment:
Sure, I used the bool because I was trying to delegate as much as possible of the business logic to ProgramService rather than having it in the controller but it's very minor logic. Switched it to ProgramType.

Reply:
mmm I see that now. I think either way is fine then, up to you